azure-steps-windows.yml

```diff
@@ -24,43 +24,20 @@ steps:
   displayName: 'Install utilities'
 
 - powershell: |
-    $ErrorActionPreference = "Stop"
-    mkdir  C:/opt/openblas/openblas_dll
-    mkdir  C:/opt/32/lib/pkgconfig
-    mkdir  C:/opt/64/lib/pkgconfig
-    $target=$(python -c "import tools.openblas_support as obs; plat=obs.get_plat(); ilp64=obs.get_ilp64(); target=f'openblas_{plat}.zip'; obs.download_openblas(target, plat, ilp64);print(target)")
-    unzip -o -d c:/opt/ $target
-    echo "##vso[task.setvariable variable=PKG_CONFIG_PATH]c:/opt/64/lib/pkgconfig"
-    copy C:/opt/64/bin/*.dll C:/opt/openblas/openblas_dll
-  displayName: 'Download / Install OpenBLAS'
-
-- powershell: |
-    # Note: ensure the `pip install .` command remains the last one here, to
-    # avoid "green on failure" issues
-    python -c "from tools import openblas_support; openblas_support.make_init('numpy')"
+    # Note: ensure the `pip install .` command remains the last one here,
+    # to avoid "green on failure" issues
     If ( Test-Path env:DISABLE_BLAS ) {
         python -m pip install . -v -Csetup-args="--vsenv" -Csetup-args="-Dblas=none" -Csetup-args="-Dlapack=none" -Csetup-args="-Dallow-noblas=true"
     }
     elseif ( Test-Path env:NPY_USE_BLAS_ILP64 ) {
-        python -m pip install . -v -Csetup-args="--vsenv" -Csetup-args="-Duse-ilp64=true" -Csetup-args="-Dblas-symbol-suffix=64_"
+        python -m pip install scipy-openblas64
+        spin build --with-scipy-openblas=64
```

Comment:
Moving `pip` invocations to `spin` should be a short-term thing, it's fine with me on Azure because that's so hard to debug. But can you please comment out the original line instead of removing it?

More importantly: I don't think this will work, because you removed the other CLI flags and they are needed. Can you instead do:
```
python -m pip install scipy-openblas64
export PKG_CONFIG_PATH=...  # whatever is the right path here
python -m pip install . -v -Csetup-args="--vsenv" -Csetup-args="-Duse-ilp64=true" -Csetup-args="-Dblas-symbol-suffix=64_"
```
?

Reply:
`spin build` will add the meson flags when invoked with `--with-scipy-openblas=64` (in addition to creating the `openblas.pc` file, modifying `_distributor_init.py`, and adding the proper `PKG_CONFIG_PATH`).